ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityActivityBlock.ts

```diff
@@ -0,0 +1,20 @@
+import { ActivityBlock } from './activityBlock';
+import { BlockType } from './blockType';
+
+enum ChildActivityRendering {
+    Embedded = 'embedded',
+    Modal = 'modal'
+}
+
+export class ActivityActivityBlock extends ActivityBlock {
+    public title: string;
+    public renderHint: ChildActivityRendering;
+    public activityCode: string;
+    public allowMultiple: boolean;
+    public addButtonText: string;
+    public instances: any[];
```

Comment:
```suggestion
    public instances: ActivityInstance[];
```
Also, pay attention that you need to update `ActivityInstance` interface, it contains a new `canDelete` property.
https://github.com/broadinstitute/ddp-study-server/blob/develop/pepper-apis/docs/specification/src/components/schemas.yml#L193

Reply:
Thsnk you! 
I was just confused that ActivityInstance and Activity.Summary are a bit different things.